fuzzers/Makefile

```diff
@@ -127,11 +127,41 @@ $(eval $(call fuzzer,100-dsp-mskpat,005-tilegrid))
 quick:
 	$(MAKE) QUICK=Y
 
-# part_only runs the fuzzers required for supporting additional parts.
-# Note: In theory this includes 005-tilegrid, but there isn't support for
-# multiple tilegrid's per family at this time.
-part_only:
+# roi_only runs the fuzzers required for supporting additional parts when building
+# an roi harness.
+roi_only:
 	+$(MAKE) -C 001-part-yaml run
 	+$(MAKE) -C 075-pins run
 
-.PHONY: all clean clean_fuzzers clean_logs quick part_only
+# Part only targets
+# -----------------
+
+part_only_yaml:
```

Comment:
Why is this written to use recursive make rather than targets with dependencies?

Reply:
Fixed